Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -253,7 +261,7 @@ else {
         Write-Host "ReleaseNotes:"
         Write-Host $releaseNotes
 
-        if (!(CommitFromNewFolder -serverUrl $serverUrl -commitMessage "Update AL-Go System Files" -branch $branch)) {
+        if (!(CommitFromNewFolder -serverUrl $serverUrl -commitMessage "Update AL-Go System Files - $templateSha" -branch $branch)) {
```

Comment:
Can we also include the branch name in the title?
E.g. "[main] Update AL-Go System Files - a1b2c3"

Otherwise, it looks a bit annoying when we run the workflow on multiple branches:
![image](https://github.com/microsoft/AL-Go/assets/43066499/6893a506-df34-40af-a6f2-3e49e9b9330f)

Reply:
good idea